dev/VSIX/Extension/Cs/Dev17/WizardImplementation.cs

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License
+
+using EnvDTE;
+using Microsoft.VisualStudio.ComponentModelHost;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.TemplateWizard;
+using Microsoft.VisualStudio.Threading;
+using NuGet.VisualStudio;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+
+namespace WindowsAppSDK.Cs.Extension
```

Comment:
I'd go ahead and change NS now - maybe something like "WindowsAppSDK.TemplateUtilities" ?